File: output/StarCoder/codenet/Python/Java/s964794058.java
import java.util.Scanner;

public class s964794058 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int memTime = sc.nextInt();
        int[] memLate = new int[memTime];
        for (int i = 0; i < memTime; i++) {
            memLate[i] = sc.nextInt();
        }
        int highLate = 0;
        int[] colorList = new int[memTime];
        for (int i = 0; i < memTime; i++) {
            int member = memLate[i];
            if (member / 400 >= 8) {
                highLate++;
            } else if (member / 400 == colorList[i]) {
                continue;
            } else {
                colorList[i] = member / 400;
            }
        }
        int maxColor = colorList.length + highLate;
        int minColor = colorList.length;
        if (minColor == 0) {
            minColor = 1;
        }
        System.out.println(minColor + " " + maxColor);
    }
}

File: output/ChatGPT/codenet/Python/C++/s964794058.cpp
int main() {
    int memTime;
    cin >> memTime;
    int memLate[memTime];
    for (int i = 0; i < memTime; i++) {
        cin >> memLate[i];
    }
    vector<int> colorList;
    int memColor = 0;
    int highLate = 0;
    for (int color = 0; color < memTime; color++) {
        int member = memLate[color];
        if (member / 400 >= 8) {
            highLate++;
        } else if (find(colorList.begin(), colorList.end(), member / 400) == colorList.end()) {
            colorList.push_back(member / 400);
        }
    }
    int maxColor = colorList.size() + highLate;
    int minColor = colorList.size();
    if (minColor == 0) {
        minColor = 1;
    }
    cout << minColor << " " << maxColor << endl;
    return 0;
}